staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -102,16 +109,20 @@ func respLoggerFromContext(req *http.Request) *respLogger {
 	return nil
 }
 
-// newLogged turns a normal response writer into a logged response writer.
-func newLogged(req *http.Request, w http.ResponseWriter) *respLogger {
+func newLoggedWithStartTime(req *http.Request, w http.ResponseWriter, startTime time.Time) *respLogger {
```

Comment:
unrelated refactor?

Reply:
Yes, will open a follow up PR. the httplogger should use the `RequestReceivedTimestamp` from the request context. Then we will have both audit and httplog print the same exact timestamp.